requirements.txt

```diff
@@ -2,9 +2,13 @@
 numpy
 scipy
 matplotlib
-mne
+PyQt5>=5.12
 qtpy
 scooby
-PyQt5
-pyqtgraph
+qtpy
```

Comment:
qtpy appears twice

Reply:
so much for the double-check.. I did look at this file for a solid minute 🙈 and still missed it. Thanks!